bp-compliments.php

```diff
@@ -28,18 +28,32 @@
  *
  * @since 0.0.1
  * @package BuddyPress_Compliments
+ *
+ * @global object $bp BuddyPress instance.
+ * @global object $wpdb WordPress db object.
  */
 function bp_compliments_init() {
     global $wpdb, $bp;
     // get plugin version from plugin data.
     $plugin_data = get_plugin_data( __FILE__ );
```

Comment:
Giri, this is not the best way to get the version number, the function get_plugin_data will have to read the file contents just simply to get the version number when you could simply just write it again in the file, please don't ever do it this way.